src/main/scala/fundamentals/level01/IntroExercises.scala

```diff
@@ -5,13 +5,19 @@ package fundamentals.level01
   * Level 1 focuses on basic Scala, including syntax and especially programming with functions.
   * We will go through a few exercises to familiarise ourselves with writing Scala.
   *
-  * What's a function?
-  *
-  * A function takes inputs and returns an output.
-  * It should always return the same output given the same inputs.
+  * An object:
+  * - An object is a class that has exactly one instance (a singleton). It is also a value.
   */
 object IntroExercises {
 
+  /**
+    * What's a function?
+    *
+    * A function takes inputs and returns an output.
+    * Question: should a function always return the same output given the same inputs?
+    *
+    */
```

Comment:
Maybe we can have both.

"It should always return the same output given the same input" (I removed the plural in `input`. The original was a typo.)

And as a presenter, you can ask: 

1. "Can two inputs give the same output?"
2. "How about the other way round? Can one input give two outputs?"